src/controllers/users.js

```diff
@@ -31,4 +32,22 @@ const editUsername = async (req, res) => {
 
 };
 
-module.exports = { getUsers, getUserById, editUsername, /* addUser, deleteUser */ }
\ No newline at end of file
+const addUser = async (req, res) => {
+	await Users.create(req.query);
```

Comment:
Have this endpoint was tested, because I'm curious how params were passed here?
In POST method params should be sent in request `body` instead of URL `query`

Reply:
It works, i tested many times

Reply:
Look at similar comment: https://github.com/ita-social-projects/EventsExpress_Node_js/pull/11#discussion_r841210318